src/subscriptions.jl

```diff
@@ -137,6 +138,8 @@ function open_subscription(fn::Function,
             end
         end
     end
+    # We can't throw errors from the ws handle function in HTTP 1.0, as they get digested.
```

Comment:
```suggestion
    # We can't throw errors from the ws handle function in HTTP.jl 1.0, as they get digested.
```

Just to make it clear it's referring to Julia package rather than HTTP protocol